<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlignmentWidget</class>
 <widget class="QWidget" name="AlignmentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>165</width>
    <height>173</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="alignWithOthersCB">
     <property name="text">
      <string>Match size with other pages</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="panelAlignment" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>4</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="alignmentLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <item row="0" column="0">
         <widget class="QToolButton" name="alignTopLeftBtn">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="icon">
           <iconset resource="../../../../app/resources/resources.qrc">
            <normaloff>:/icons/stock-gravity-north-west-24.png</normaloff>:/icons/stock-gravity-north-west-24.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QToolButton" name="alignBottomBtn">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="icon">
           <iconset resource="../../../../app/resources/resources.qrc">
            <normaloff>:/icons/stock-gravity-south-24.png</normaloff>:/icons/stock-gravity-south-24.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QToolButton" name="alignTopBtn">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="icon">
           <iconset resource="../../../../app/resources/resources.qrc">
            <normaloff>:/icons/stock-gravity-north-24.png</normaloff>:/icons/stock-gravity-north-24.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QToolButton" name="alignRightBtn">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="icon">
           <iconset resource="../../../../app/resources/resources.qrc">
            <normaloff>:/icons/stock-gravity-east-24.png</normaloff>:/icons/stock-gravity-east-24.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QToolButton" name="alignCenterBtn">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="icon">
           <iconset resource="../../../../app/resources/resources.qrc">
            <normaloff>:/icons/stock-gravity-center-24.png</normaloff>:/icons/stock-gravity-center-24.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="alignTopRightBtn">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="icon">
           <iconset resource="../../../../app/resources/resources.qrc">
            <normaloff>:/icons/stock-gravity-north-east-24.png</normaloff>:/icons/stock-gravity-north-east-24.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QToolButton" name="alignLeftBtn">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="icon">
           <iconset resource="../../../../app/resources/resources.qrc">
            <normaloff>:/icons/stock-gravity-west-24.png</normaloff>:/icons/stock-gravity-west-24.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QToolButton" name="alignBottomLeftBtn">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="icon">
           <iconset resource="../../../../app/resources/resources.qrc">
            <normaloff>:/icons/stock-gravity-south-west-24.png</normaloff>:/icons/stock-gravity-south-west-24.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QToolButton" name="alignBottomRightBtn">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="icon">
           <iconset resource="../../../../app/resources/resources.qrc">
            <normaloff>:/icons/stock-gravity-south-east-24.png</normaloff>:/icons/stock-gravity-south-east-24.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>4</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="panelAdvancedAlignment" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_18">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>4</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="AlignmentComboBox" name="cbAutoMagnet">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select alignment automatically&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="statusTip">
         <string>Automatically select alignment based on how close content to page borders</string>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
        <item>
         <property name="text">
          <string>Disabled</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../app/resources/resources.qrc">
           <normaloff>:/icons/auto-magnet-none.png</normaloff>:/icons/auto-magnet-none.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Auto-magnet both axes</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../app/resources/resources.qrc">
           <normaloff>:/icons/auto-magnet.png</normaloff>:/icons/auto-magnet.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Auto-magnet vertically</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../app/resources/resources.qrc">
           <normaloff>:/icons/auto-magnet-ver.png</normaloff>:/icons/auto-magnet-ver.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Auto-magnet horizontally</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../app/resources/resources.qrc">
           <normaloff>:/icons/auto-magnet-hor.png</normaloff>:/icons/auto-magnet-hor.png</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="AlignmentComboBox" name="cbOriginalProp">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="toolTip">
         <string>Place proportionally to position on original scan</string>
        </property>
        <property name="statusTip">
         <string>Place proportionally to position on original scan</string>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
        <item>
         <property name="text">
          <string>Disabled</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../app/resources/resources.qrc">
           <normaloff>:/icons/gimp-center-none.png</normaloff>:/icons/gimp-center-none.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Original proportions all axes</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../app/resources/resources.qrc">
           <normaloff>:/icons/stock-center-24.png</normaloff>:/icons/stock-center-24.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Original proportions vertically</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../app/resources/resources.qrc">
           <normaloff>:/icons/stock-center-24-ver.png</normaloff>:/icons/stock-center-24-ver.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Original proportions horizontally</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../app/resources/resources.qrc">
           <normaloff>:/icons/stock-center-24-hor.png</normaloff>:/icons/stock-center-24-hor.png</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnResetAdvAlignment">
        <property name="toolTip">
         <string>Reset advanced alignment</string>
        </property>
        <property name="statusTip">
         <string>Reset advanced alignment options</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="icon">
         <iconset resource="../../../../app/resources/resources.qrc">
          <normaloff>:/icons/gimp-undo-history.png</normaloff>:/icons/gimp-undo-history.png</iconset>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_17">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>4</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AlignmentComboBox</class>
   <extends>QComboBox</extends>
   <header>filters/page_layout/alignmentwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>alignWithOthersCB</tabstop>
  <tabstop>alignTopLeftBtn</tabstop>
  <tabstop>alignTopBtn</tabstop>
  <tabstop>alignTopRightBtn</tabstop>
  <tabstop>alignLeftBtn</tabstop>
  <tabstop>alignCenterBtn</tabstop>
  <tabstop>alignRightBtn</tabstop>
  <tabstop>alignBottomLeftBtn</tabstop>
  <tabstop>alignBottomBtn</tabstop>
  <tabstop>alignBottomRightBtn</tabstop>
  <tabstop>cbAutoMagnet</tabstop>
  <tabstop>cbOriginalProp</tabstop>
  <tabstop>btnResetAdvAlignment</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../app/resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
